src/main/java/seedu/address/model/delivery/Date.java

```diff
@@ -10,8 +10,9 @@
  * Represents a Date in the address book.
  */
 public class Date implements Comparable<Date> {
+    public static final String FORMAT = "yyyy-MM-dd";
```

Comment:
If we're using this form of capitalisation, we may have to update UG (feature) and DG (use cases).

Reply:
Resolved. I replaced the UG but DG don't have when I try to find.